scripts/query_collection.R

```diff
@@ -80,8 +80,9 @@ for (page in 1:last){
 
     # Keep only those with homepages and descriptions
     filter(!(is.na(homepage)), homepage != "",!(is.na(description)))
-  
-  full_repo_df <- rbind(full_repo_df, repo_df)
+
+  full_repo_df <- rbind(full_repo_df, repo_df) %>%
+    dplyr::arrange(name)
```

Comment:
@avahoffman do you prefer the arrange step goes in the next section?

Reply:
nope this is good!